app/root.module.css

```
@@ -0,0 +1,9 @@
+.root {
+  width: 50svw;
+  min-width: 20rem;
```

```suggestion
  max-width: 60rem;
```
小さい画面のときに半端な余白が出てしまうので, このくらいの最大幅でどうでしょうか (本来はグリッドレイアウトでコンテナ側を調節するようにしたいですが, 今のところはこれでもよさそうです).